$MirOS: ports/misc/cksfv/patches/patch-src_print_c,v 1.1.7.1 2005/03/18 15:50:25 tg Exp $
--- src/print.c.orig	Fri Jul  6 06:32:46 2001
+++ src/print.c	Fri Dec 30 17:56:03 2005
@@ -26,13 +26,13 @@
 
 void pnsfv_head()
 {
-  clock_t       clock;
+  time_t	clock;
   struct tm     *timeinfo;
   
   clock = time(NULL);
   timeinfo = localtime(&clock);
   printf("; Generated by cksfv v%s on ", VERSION);
-  printf("%02d-%02d-%02d at %02d:%02d.%02d\n", timeinfo->tm_year+1900,
+  printf("%02d-%02d-%02d at %02d:%02d.%02d\n", (int)timeinfo->tm_year+1900,
          timeinfo->tm_mon, timeinfo->tm_mday, timeinfo->tm_hour,
          timeinfo->tm_min, timeinfo->tm_sec);
   printf("; Written by Bryan Call <bc@fodder.org> ");
@@ -54,7 +54,7 @@ void pfileinfo(char **argv)
       printf(";%13d  %02d:%02d.%02d %02d-%02d-%02d %s\n",
              (int)sb.st_size, timeinfo->tm_hour, timeinfo->tm_min,
              timeinfo->tm_sec,
-             timeinfo->tm_year+1900, timeinfo->tm_mon, timeinfo->tm_mday, fn);
+             (int)timeinfo->tm_year+1900, timeinfo->tm_mon, timeinfo->tm_mday, fn);
     }
   }
 }
